test/testutils/util.go

```diff
@@ -392,6 +392,21 @@ func HasLWSEnvVarsPopulated(pod corev1.Pod) bool {
 	return hasAllEnvVarPopulated(pod, []string{leaderworkerset.LwsLeaderAddress, leaderworkerset.LwsGroupSize, leaderworkerset.LwsWorkerIndex})
 }
 
+func CheckSizeAnnotation(pod corev1.Pod, size int) error {
```

Comment:
Can we make this move general like`CheckAnnotation(pood, key, value)`

Reply:
Sure !